airbyte-integrations/connectors/destination-postgres/src/main/kotlin/io/airbyte/integrations/destination/postgres/client/PostgresAirbyteClient.kt

```diff
@@ -121,39 +118,108 @@ class PostgresAirbyteClient(
             log.info { "Added columns: $addedColumns" }
             log.info { "Deleted columns: $deletedColumns" }
             log.info { "Modified columns: $modifiedColumns" }
-            sqlGenerator
-                .matchSchemas(tableName, addedColumns, deletedColumns, modifiedColumns, columnsInDb)
-                .forEach { execute(it) }
+
+            execute(sqlGenerator.matchSchemas(
```

Comment:
Lots of args, but if we want to do everything in one same transaction we have to do something like this.